cli/command/container/stats.go

```diff
@@ -156,7 +156,7 @@ func runStats(dockerCli command.Cli, opts *statsOptions) error {
 		// Start a short-lived goroutine to retrieve the initial list of
 		// containers.
 		getContainerList()
-		
+
```

Comment:
This is really odd; last PR was https://github.com/docker/cli/pull/2785, and that commit doesn't show the whitespace 🤔 ??? https://github.com/docker/cli/commit/53e2110fa35590f7f4aa090a77a58958fc8f94c4

Reply:
Checking the "blame" view doesn't show the whitespace; https://github.com/docker/cli/blame/b07e921289f7d3ec80e91636d7979298975ea210/cli/command/container/stats.go#L159

<img width="1222" alt="Screenshot 2020-11-10 at 17 53 43" src="https://user-images.githubusercontent.com/1804568/98705223-c5b83500-237d-11eb-967b-638443e9345a.png">

Reply:
Also checking the diff on the PR, and it doesn't show the whitespace; I checked if I had "hide whitespace changes" enabled, but I don't; https://github.com/docker/cli/pull/2785/files

<img width="546" alt="Screenshot 2020-11-10 at 17 55 34" src="https://user-images.githubusercontent.com/1804568/98705477-0c0d9400-237e-11eb-8b9d-d1429f3e32fb.png">